File: engine/model/src/main/scala/tools/packet/buff/GiveFinalAttack.scala
package tools.packet.buff

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class GiveFinalAttack(private var _skillId: Int, private var _time: Int) extends PacketInput {
  def skillId: Int = _skillId

  def time: Int = _time

  override def opcode(): SendOpcode = SendOpcode.GIVE_BUFF
}

File: engine/engine-model/src/main/java/tools/packet/buff/GiveFinalAttack.java
package tools.packet.buff;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record GiveFinalAttack(Integer skillId, Integer time) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.GIVE_BUFF;
   }
}
